docs/guides/index.md

```diff
@@ -527,15 +527,16 @@ Events][animatingonevents] below.
 
 In JavaScript, we grab the box with [`querySelector`][queryselector], use
 [`addEventListener`][addeventlistener], and then [`setAttribute`][setattribute]
-to make the box grow its scale when its hovered over. Note that A-Frame adds
+to make the box grow its scale when its hovered over. The code must appear after
```

Comment:
Can we separate this in 2 PRs. One with the typo that I will quickly merge and the other with the clarification and @ngokevin can evaluate